lib/fastlane/plugin/notarize/version.rb

```diff
@@ -1,5 +1,5 @@
 module Fastlane
   module Notarize
-    VERSION = '0.2.1'
+    VERSION = '0.2.2'
```

Comment:
No need to bump the version in this branch!

Reply:
Fixed